solr/jmx-merge-solr7plus.yml

```diff
@@ -0,0 +1,342 @@
+type: jmx
+require:
+  - className: com.sematext.spm.client.solr.SolrVersionCheck
+    values: 7-*
+
+observation:
+  - name: merge_errors_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=INDEX,scope=merge,name=errors
+
+    metric:
+      - name: index.merge.errors
+        source: Count
+        type: counter
+        label: merge errors
+        description: merge errors
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: merge_minor_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=INDEX,scope=merge,name=minor
+
+    metric:
+      - name: index.merge.minor
+        source: Count
+        type: counter
+        label: minor merges
+        description: minor merges
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: merge_minor_running_docs_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=INDEX,scope=merge,name=minor.running.docs
+
+    metric:
+      - name: index.merge.minor.running.docs
+        source: Value
+        type: counter
+        label: docs merged in minor merges
+        description: docs merged in minor merges
+
+    tag:
+      - name: solr.collection
+        value: ${collection}
+
+      - name: solr.shard
+        value: ${shard}
+
+      - name: solr.replica
+        value: ${replica}
+
+      - name: solr.core
+        value: func:ConcatUsingString(_,collection,shard,replica)
+
+  - name: merge_minor_running_segments_solr7_cloud
+    metricNamespace: solr
+    objectName: solr:dom1=core,dom2=${collection},dom3=${shard},dom4=${replica},category=INDEX,scope=merge,name=minor.running.segments
+
+    metric:
+      - name: index.merge.minor.running.segments
```

Comment:
Is this metric about "merges that are currently in progress"?

Reply:
That's right! Which is why this is a bit tricky.

Reply:
Then I would suggest reflecting that in metric label/description. Also, it can't be of counter type in that case, it has to be a gauge. And it is probably the same for other metrics that have `running` in their name.

Reply:
Shoot. You're right. Let me adjust.